server/lib/RouteResolver.js

```diff
@@ -11,8 +11,26 @@ const _ = require('lodash');
 let app;
 let fixturesDir;
 
+const validateResponse = function validateResponse(response) {
+  const responseProps = [];
+  const responsePropTypes = ['fixture', 'filePath', 'html', 'json', 'text'];
```

Comment:
Yeah, that makes sense, I agree with not naming things like React does. It seemed to just flow when I was typing. 

I don't mind what you have with payloads, it kind of looks more like a `responseTypes` or a `contentTypes` to me. But `payload` also makes sense. I'm fine either way.